src/components/SecuritySection.vue

```diff
@@ -0,0 +1,203 @@
+<!--
+  - SPDX-FileCopyrightText: 2022 Carl Schwan <carl@carlschwan.eu>
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+  -->
+
+<template>
+	<SettingsSection :title="t('end_to_end_encryption', 'End to End Encryption')"
+		:description="encryptionState">
+		<Button :disabled="!hasKey" type="warning" @click="startResetProcess()">
+			{{ t('end_to_end_encryption', 'Reset End to End Encryption') }}
+		</Button>
+
+		<div v-if="shouldDisplayWarning && hasKey" class="notecard warning" role="alert">
+			<p>{{ t('end_to_end_encryption', 'Please read carefully before resetting your end-to-end encryption keys') }}</p>
```

Comment:
```suggestion
			<p><strong>{{ t('end_to_end_encryption', 'Please read carefully before resetting your end-to-end encryption keys') }}</strong></p>
```
Bold just to be clear here?